### ISSUE TITLE
regression:: achavi query no longer working on new_features branch

### ISSUE THREAD

**Issue by mmd-osm** [drolbr/Overpass-API#412]
The following achavi query no longer returns any results on the new features branch: http://overpass-turbo.eu/s/qp1

Tested on https://github.com/drolbr/Overpass-API/commit/97a7c1bef7b4c7bca137acc4f043dc3edf7e89b0

**Comment by mmd-osm** [comment]
Those two lines are buggy:

```
 since = Timestamp(attributes["since"]).timestamp;
until = Timestamp(attributes["until"]).timestamp;
```
Since since and now are initialized with `NEW`, those values may not be overwritten in case of  attributes["since"] = "auto"; and attributes["until"] = "auto";

The new Timestamp method has no idea about this special logic and happily sets both `since` and `until` to 0 instead.

**Comment by drolbr** [comment]
Thank you for the regression notification.

The crucial part here is that there is no testing for the use case. Adding tests has just revealed a couple of related bugs which are now fixed.

All fixed in 2e95d9149d42ae1fa4d26d8962c10483833da574.

**Comment by mmd-osm** [comment]
Great, thanks a lot for the fix. I have merged your fix in my continuous integration branch and deployed it to //dev.overpass-api.de/api_mmd/ for testing now.
